packages/@lwc/compiler/src/options.ts

```diff
@@ -139,6 +148,15 @@ function validateOptions(options: TransformOptions) {
         );
     }
 
+    if (!isUndefined(options.stylesheetConfig) && !alreadyWarnedOnStylesheetConfig) {
+        alreadyWarnedOnStylesheetConfig = true;
+
+        // eslint-disable-next-line no-console
+        console.warn(
+            `"stylesheetConfig" property is deprecated. The value doesn't impact the compilation and can safely be removed.`
+        );
+    }
+
```

Comment:
We can combine this `if` condition with the one below it

Reply:
@nolanlawson updated, removed the `validateStylesheetConfig` as it doesn't really serve any purpose anymore.

Reply:
Nice! More red. 🎉